userspace/falco/outputs_http.cpp

```diff
@@ -18,104 +18,105 @@ limitations under the License.
 #include "logger.h"
 #include "banned.h" // This raises a compilation error when certain functions are used
 
-void falco::outputs::output_http::output(const message *msg)
+#define CHECK_RES(fn) res = res == CURLE_OK ? fn : res
+
+static size_t noop_write_callback(void *contents, size_t size, size_t nmemb, void *userp)
+{
+	// We don't want to echo anything. Just return size of bytes ignored
+	return size * nmemb;
+}
+
+bool falco::outputs::output_http::init(const config& oc, bool buffered, const std::string& hostname, bool json_output, std::string &err)
 {
-	CURL *curl = NULL;
+	falco::outputs::abstract_output::init(oc, buffered, hostname, json_output, err);
```

Comment:
Should the return value of the parent `init` be checked?

I believe yes since `falco::outputs::abstract_output::init` can change in the future and start returning errors in some cases.

Reply:
Makes sense :) thanks!

Reply:
Fixed!